addons/arsenal/functions/fnc_buttonImport.sqf

```diff
@@ -15,102 +15,72 @@
 
 params ["_display"];
 
-private _data = call (compile copyFromClipboard);
+// Can be either a singular loadout or an array of loadouts
+private _extendedLoadout = call compile copyFromClipboard;
 
-if (isNil "_data" || {!(_data isEqualType [])}) exitWith {
-    [_display, localize LSTRING(importFormatError)] call FUNC(message);
+// If error, exit
+if (isNil "_extendedLoadout" || {!(_extendedLoadout isEqualType [])}) exitWith {
+    [_display, LLSTRING(importFormatError)] call FUNC(message);
 };
 
 if (GVAR(shiftState) && {is3DEN}) then {
-
+    // Supports CBA extended loadout or getUnitLoadout arrays
     {
         if (
-                count _x == 2 &&
-                {_x select 0 isEqualType ""} &&
-                {_x select 0 != ""} &&
-                {_x select 1 isEqualType []} &&
-                {count (_x select 1) == 10}
+            count _x == 2 &&
+            {(_x select 0) isEqualType ""} &&
```

Comment:
There's no need to surround with parenthesis like this, but it does make it more readable.